### PR TITLE
aria2 continue interrupted downloads

### PR DESCRIPTION
**Pull request by candrapersada**
# Pull Request Template

## Type of change

- [x] Bug fix
- [ ] Feature
- [ ] Documentation update

## Description

*ramble here*

## Checklist

- [ ] any anime playing
- [ ] bumped version
---
- [ ] next, prev and replay work
- [ ] `-c` history and continue work
- [x] `-d` downloads work
- [ ] `-s` syncplay works
- [ ] `-q` quality works
- [ ] `-v` vlc works
- [ ] `-e` select episode works
- [ ] `-r` range selection works
- [ ] `--dub` both work
- [ ] all providers return links (not necessarily on a single anime, use debug mode to confirm)
---
- [ ] `-h` help info is up to date
- [ ] Readme is up to date
- [ ] Man page is up to date

## Additional Testcases

- The safe bet: One Piece
- Episode 0: Saenai Heroine no Sodatekata ♭
- Unicode: Saenai Heroine no Sodatekata ♭
- Non-whole episodes: Tensei shitara slime datta ken (ep. 24.5, ep. 24.9)

### CODE REVIEW

**Comment by port19x** [comment]
Please explain the motivations for turning this option off by default

**Comment by candrapersada** [comment]
in order to continue a failed download without restarting the download from the beginning with The new file name has a dot and a number(1..9999) appended after the name.

**Comment by port19x** [comment]
In that case, the -c flag may be more appropriate.
-c continues the file download for partially downloaded files.
-c doesn't change behaviour of fresh downloads either.

File: ani-cli [hunk 1]
```diff
@@ -204,7 +204,7 @@ update_history() {
 download() {
 	case $1 in
 	*m3u8*) ffmpeg -loglevel error -stats -i "$1" -c copy "$download_dir/$2.mp4" ;;
-	*) aria2c --check-certificate=false --summary-interval=0 -x 16 -s 16 "$1" --dir="$download_dir" -o "$2.mp4" --download-result=hide ;;
+	*) aria2c --check-certificate=false --auto-file-renaming=false --summary-interval=0 -x 16 -s 16 "$1" --dir="$download_dir" -o "$2.mp4" --download-result=hide ;;
```
Comment (port19x) [inline ani-cli:207]:
```suggestion
	*) aria2c --check-certificate=false --continue --summary-interval=0 -x 16 -s 16 "$1" --dir="$download_dir" -o "$2.mp4" --download-result=hide ;;
```

**Review by port19x — CHANGES_REQUESTED** [review]

**Review by port19x — APPROVED** [review]

**Comment by candrapersada** [comment]
image

**Pull request merged**